ios/src/platform_ios.mm

```diff
@@ -153,20 +153,23 @@ bool isContinuousRendering() {
 
     for (id fallback in fallbacks) {
 
-        UIFont* font = [UIFont fontWithName:fallback size:1.0];
+        handles.emplace_back([fallback]() {
 
-        size_t dataSize = 0;
-        auto cdata = loadUIFont(font, &dataSize);
+             __block std::vector<char> data;
 
-        if (!cdata) { continue; }
+             dispatch_sync(dispatch_get_main_queue(), ^{
```

Comment:
yes, that seems sensible. it's my first lines objective-c code, so I would have certainly asked for some testing and advice :)